crates/compiler/load_internal/src/file.rs

```diff
@@ -3442,6 +3475,27 @@ fn load_platform_module<'a>(
                     "expected platform/package module, got App with header\n{:?}",
                     header
                 ))),
+                Ok((
+                    ast::Module {
+                        header: ast::Header::Package(header),
+                        ..
+                    },
+                    parser_state,
+                )) => {
+                    let (_, _, package_module_msg) = build_package_header(
+                        arena,
+                        Some(shorthand),
+                        false, // since we have an app module ID, the app module must be the root
```

Comment:
Could maybe assign the `false` to an identifier to give it meaning too.

Reply:
Good call! I'll revise this in the next PR. 👍